tutorials/appengine-dropwizard/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Deploying Apache Dropwizard applications on Google App Engine
+description: Learn how to deploy Apache Dropwizard applications on Google App Engine Flexible environment using custom runtime.
+author: agentmilindu
+tags: App Engine, Aapche, Dropwizard
+date_published: 2017-10-29
+---
+This tutorial shows how to deploy an [Apache Dropwizard][dropwizard] application
+on Google App Engine flexible environment.
+
+
+##Overview
+
+[Apache Dropwizard][dropwizard] is a Java framework for developing RESTful 
+web services which support for configuration management, application metrics,
+ logging, operational tools, etc out of the box. 
+
+Deploying Dropwizard applications on Google App Engine
+ Flexible Environment is not straightforward as it needs passing arguments 
```

Comment:
"not straightforward", while probably accurate, comes across as quite negative. Perhaps something like, "Deploying Dropwizard applications on Google App Engine Flexible Environment can be tricky, because it requires you to pass arguments..."